src/RpcClient/WalletAPI.cs

```diff
@@ -156,6 +157,23 @@ public Transaction Transfer(UInt160 scriptHash, KeyPair from, UInt160 to, BigInt
             return transaction;
         }
 
+        /// <summary>
+        /// Transfer NEP5 token from multi-sig account
+        /// </summary>
+        /// <param name="scriptHash">contract script hash</param>
+        /// <param name="m">multi-sig min signature count</param>
+        /// <param name="pubKeys">multi-sig pubKeys</param>
+        /// <param name="keys">sign keys</param>
+        /// <param name="to">to account</param>
+        /// <param name="amountInteger">transfer amount</param>
+        /// <returns></returns>
+        public Transaction Transfer(UInt160 scriptHash, int m, ECPoint[] pubKeys, KeyPair[] keys, UInt160 to, BigInteger amountInteger)
```

Comment:
Yes, we may wrap async methods for code reuse. https://github.com/chenquanyu/neo-modules/blob/add-async-methods/src/RpcClient/RpcClient.Async.cs

Reply:
The underlying HttpClient doesn't expose sync versions of Get, Post, etc since Http requests are inherently async. Shouldn't all the RpcWrapper methods follow the same strategy? Why are there sync methods for any of these RPC wrapper calls?

Reply:
I think we can add async methods after the discussion in #209.